x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/ner/ner.tsx

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import React, { FC, useState, useMemo } from 'react';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiSpacer, EuiTextArea, EuiButton, EuiTabs, EuiTab } from '@elastic/eui';
+
+import { useMlApiContext } from '../../../../../contexts/kibana';
+import { NerInference } from './ner_inference';
+import type { FormattedNerResp } from './ner_inference';
+import { NerOutput } from './ner_output';
+import { MLJobEditor } from '../../../../../jobs/jobs_list/components/ml_job_editor';
+import { extractErrorMessage } from '../../../../../../../common/util/errors';
+import { ErrorMessage } from '../../inference_error';
+import { OutputLoadingContent } from '../../output_loading';
+
+interface Props {
+  model: estypes.MlTrainedModelConfig;
+}
+
+enum TAB {
+  TEXT,
+  RAW,
+}
+
+export const NerModel: FC<Props> = ({ model }) => {
+  const { trainedModels } = useMlApiContext();
+
+  const ner = useMemo(() => new NerInference(trainedModels, model), [trainedModels, model]);
+
+  // const [inputText, setInputText] = useState('');
+  const [inputText, setInputText] =
+    useState(`Hello, my name is James and i live in Hersham, Surrey with my wife Kate, and my children Jonathan and Emily.
```

Comment:
```suggestion
    useState(`Hello, my name is James and I live in Hersham, Surrey with my wife Kate, and my children Jonathan and Emily.
```

Reply:
Updated in https://github.com/elastic/kibana/commit/53125e5673d7d6d6e3cfd807c21910b031c2d035